src/components/Button/KeyBox/KeyBox.js

```diff
@@ -37,30 +45,43 @@ const StyledButton = styled(Box)`
   width: 100%;
   border-radius: 8px;
   padding: 8px;
-  color: ${({ theme }) => theme.color.primary.default};
-  background-color: ${({ theme }) => theme.color.primary.t.lighten5};
   height: 100%;
-
+  color: ${({ theme }) =>
+    theme?.color?.primary?.default || typeformTheme?.color?.primary?.default};
+  background-color: ${({ theme, isActive }) =>
+    isActive
+      ? theme?.color?.primary?.t?.lighten2 ||
+        typeformTheme?.color?.primary?.t?.lighten2
+      : theme?.color?.primary?.t?.lighten5 ||
+        typeformTheme?.color?.primary?.t?.lighten5};
   &:hover {
-    background-color: ${({ theme }) => theme.color.primary.t.lighten3};
+    background-color: ${({ theme }) =>
+      theme?.color?.primary?.t?.lighten3 ||
+      typeformTheme?.color?.primary?.t?.lighten3};
   }
+  ${blinkBackground}
+  ${({ isActive }) =>
+    isActive &&
+    css`
+      animation: blinkBackground 0.5s ease;
+    `}
 `
 const StyledText = styled(Text)`
   width: ${({ hasImages, phoneSmall }) =>
     hasImages ? (phoneSmall ? '100%' : '15ch') : '100%'};
   padding-left: 30px;
 `
 
-const StyledBadge = styled(Button)`
-  position: absolute;
-  border-radius: 50%;
-  height: 24px;
-  z-index: 100;
-  padding: 3px;
-  width: 24px;
-  right: 0;
-  top: -4px;
-`
+// const StyledBadge = styled(Button)`
+//   position: absolute;
```

Comment:
are you about to keep those button for the future?

Reply:
Yes